formbricks/formbricks
# feat: Added profile image delete button

## What does this PR do?

Add a button to delete your current profile picture
![image](https://github.com/formbricks/formbricks/assets/22999200/9571ecf0-7740-4c24-99e7-e100b6a6bf94)

## How should this be tested?

- Add a profile image
- Check if the delete button is visible
- Delete your profile image

## Checklist

- [x] Add a button to delete your current profile image
- [x] Add a action that will handleRemove of the profile image
- [ ] Add a function that will delete the S3 object

### Required


### Appreciated
